fabric-item-groups-v0/build.gradle

```diff
@@ -1,5 +1,5 @@
 archivesBaseName = "fabric-item-groups-v0"
-version = getSubprojectVersion(project, "0.1.12")
+version = getSubprojectVersion(project, "0.1.13")
```

Comment:
Id almost go to 0.2.0 no reason not to tbh.

Reply:
true, I'll bump to that and remove the krita generated file that got left in for some reason